code/extensions/FluentExtension.php

```diff
@@ -715,7 +715,13 @@ public function updateCMSFields(FieldList $fields)
                 }
 
                 // Highlight any translated field
-                if ($field) {
+                if($field && !$field->hasClass('LocalisedField')) {
+                    $field->addExtraClass('LocalisedField');
```

Comment:
I think this is duplicated below? That extra check you added is a good idea though.